core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -231,10 +234,11 @@ private void updateBalls(float delta) {
 			ballStaticDeadList.remove(0);
 			System.out.println("number of balls left: " + ballGraph.numberOfBalls());
 			if (ballStaticDeadList.size() == 0) {
+				GameScreen.scoreKeeper.setCurrentScore(0, ballGraph.getFreeBalls().size());
```

Comment:
Score application isn't yet done. will do it in another branch.
Will remove this line.